src/test/java/TestLIC1.java

```diff
@@ -1,85 +1,114 @@
-    Point[] pts = {a, b, c};
-    Assert.assertTrue(Decide.LIC1(pts, params));
-  }
+    /**
+     * Generates a three points on the x-axis from -2 to 2, asserts that points exactly radius1
+     * from the center yields a negative result.
+     */
+    @Test
+    public void testOnCircle() {
+        Parameters params = new Parameters(0, 0, 2, 0, 0,
+                0, 0, 0, 0, 0, 0, 0, 0, 0,
+                0, 0, 0, 0, 0);
+        Point a = new Point(0, 0);
+        Point b = new Point(0, 2);
+        Point c = new Point(0, -2);
+        Point[] pts = {a, b, c};
+        Assert.assertFalse(Decide.LIC1(pts, params));
+    }
 
-  @Test
-  public void testTriangularPoints() {
-    // Testing points where the third point can't fit into a circle drawn with diameter
-    // between the first and second points.
-    Parameters params = new Parameters(0, 0, 5, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0);
-    Point a = new Point(4, 0);
-    Point b = new Point(-4, 0);
-    Point c = new Point(0, 7);
-    Point[] pts = {a, b, c};
-    Assert.assertFalse(Decide.LIC1(pts, params));
+    /**
+     * Assert that three consecutive data points that cannot all be contained within a circle of
+     * radius1 yields a positive result.
+     */
+    @Test
+    public void testOutsideCircle() {
+        Parameters params = new Parameters(0, 0, 5, 0, 0,
+                0, 0, 0, 0, 0, 0, 0, 0, 0,
+                0, 0, 0, 0, 0);
+        Point a = new Point(0, 0);
+        Point b = new Point(10, 10);
+        Point c = new Point(1, 1);
+        Point[] pts = {a, b, c};
+        Assert.assertTrue(Decide.LIC1(pts, params));
+    }
 
-    Point d = new Point(0, 9);
-    Point[] pts2 = {a, b, d};
-    Assert.assertTrue(Decide.LIC1(pts2, params));
-  }
+    /**
+     * Testing points where the third point can't fit into a circle drawn with diameter.
+     */
+    @Test
+    public void testTriangularPoints() {
+        // between the first and second points.
+        Parameters params = new Parameters(0, 0, 5, 0, 0,
+                0, 0, 0, 0, 0, 0, 0, 0, 0,
+                0, 0, 0, 0, 0);
+        Point a = new Point(4, 0);
+        Point b = new Point(-4, 0);
+        Point c = new Point(0, 7);
+        Point[] pts = {a, b, c};
+        Assert.assertFalse(Decide.LIC1(pts, params));
 
-  @Test
-  public void testLines() {
-    // Testing points that are forming lines.
-    Parameters params = new Parameters(0, 0, 5, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0);
-    Point a = new Point(0, 0);
-    Point b = new Point(2, 3);
-    Point c = new Point(4, 6);
-    Point[] pts = {a, b, c};
-    Assert.assertFalse(Decide.LIC1(pts, params));
+        Point d = new Point(0, 9);
+        Point[] pts2 = {a, b, d};
+        Assert.assertTrue(Decide.LIC1(pts2, params));
+    }
+
+    /**
+     * Testing points forming a line.
+     * First asserting that a line that can fit into a circle of radius1 yields a negative result.
+     * Secondly the opposite.
```

Comment:
You could elaborate what you mean by "the opposite".

Reply:
Requested changes made and pushed!